recipes/theora/all/CMakeLists.txt

```
@@ -0,0 +1,131 @@
+cmake_minimum_required(VERSION 3.15)
+project(theora C)
+
+# Note that I wrote this cmake file because:
+# - the old msvc sln was very old and required upgrades each time it was built
+# - the old msvc did not support RelWithDebInfo
+# - the recipe needed to be upgraded for conan-v2, so a simpler recipe was desired
+
+find_package(Ogg REQUIRED)
```

there's no module file https://github.com/conan-io/conan-center-index/blob/6d2a713b6a0e10e81168b86f5d101f291c973da5/recipes/ogg/all/conanfile.py#L78

```suggestion
find_package(Ogg CONFIG REQUIRED)
```